src/databricks/labs/ucx/source_code/linters/files.py

```diff
@@ -38,21 +38,21 @@ def __init__(self, path: Path, source: str, language: Language):
         # using CellLanguage so we can reuse the facilities it provides
         self._language = CellLanguage.of_language(language)
 
+    @property
```

Comment:
why do we need this?

Reply:
It was already there, it was just moved up (properties before methods)